lib/Slackbot_worker.js

```diff
@@ -716,7 +717,7 @@ module.exports = function(botkit, config) {
      */
     bot.findConversation = function(message, cb) {
         botkit.debug('CUSTOM FIND CONVO', message.user, message.channel);
-        if (message.type == 'message' || message.type == 'slash_command' ||
+        if (message.type == 'direct_message' || message.type == 'ambient' || message.type == 'mention' || message.type == 'slash_command' ||
```

Comment:
1) YOUCH, can't believe direct_mention was excluded from this list for so long oops

2) I agree this should be an array and in fact it should be a customizable array.